### ISSUE TITLE
Allow for calculation when adding multiple sources

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#5098]
**Is your feature request related to a problem? Please describe.**
Team want to know a percentage of certain stories on the backlog, for instance. 

**Describe the solution you'd like**
It would be useful if there was a metric that had multiple sources configured, could make a calculation to display the end result.
Metric A has source 1 that has value 200 and source 2 that had value 30.
The metric is configured to calculate the percentage of source 2 and 1.
Because of this, the metric displays 15% in this case.

**Describe alternatives you've considered**
Calculate by hand.

**Comment by fniessink** [comment]
I don't think we will build the generic solution, because that is a very big change. However, we could make the issues metric a bit more generic maybe and have it support the percentage scale.

Can you elaborate your example? If the percentage is calculated as Y / X * 100%, what exactly are X and Y?

**Comment by Sebastiaan127001** [comment]
The two use cases as discussed:
### use case 1
Users want to know what the percentage is of issues in the Jira backlog or the past sprint that have a certain label. They have agreed with the product owner that 10% should be technical debt issues (with quality label or part of an epic).
numerator = all JIRA issues having a label named 'ABC' or all JIRA issues child of Epic ABD-123
denominator = all JIRA issues in backlog or all JIRA issues in last closed sprint

### use case 2
users want to know the percentage of successful job runs or pipeline runs in the last 21 days in GitLab
numerator = job runs or pipeline runs with status passed / success
denominator = all job runs or pipeline runs in the last 21 days
